specifyweb/context/views.py

```diff
@@ -258,6 +258,10 @@ def user(request):
     data = obj_to_data(request.specify_user)
     data['isauthenticated'] = request.user.is_authenticated
     data['available_collections'] = users_collections(connection.cursor(), request.specify_user.id)
+
+    if not hasattr(request.specify_user_agent, '_meta'):
```

Comment:
I think this should be a simple `request.specify_user_agent is None` check.

Reply:
This appears to be necessary as type(request.specify_user_agent) returns `<class 'django.utils.functional.SimpleLazyObject'>`